libvast/src/system/terminator.cpp

```diff
@@ -24,7 +25,9 @@
 namespace vast::system {
 
 template <class Policy>
-caf::behavior terminator(caf::stateful_actor<terminator_state>* self) {
+caf::behavior terminator(caf::stateful_actor<terminator_state>* self,
+                         std::chrono::seconds clean_exit_timeout,
+                         std::chrono::seconds kill_exit_timeout) {
```

Comment:
I think "grace period" is the commonly used term for the clean exit timeout.

1 second minimum timeout seems pretty long, using milli- or microsecond precision would feel slightly more future-proof.

Reply:
> I think "grace period" is the commonly used term for the clean exit timeout.

I tried to find two names for timeouts that make sense together. If we call the first `grace_period`, how would you call the second, `kill_period`?